lib/rules/web/admin_console/4.2/user_management.xyaml

```diff
@@ -88,3 +88,32 @@ check_cluster_role_reference:
     resource_name: <role_name>
     resource_link: /k8s/cluster/clusterroles/<role_name>
   action: check_resource_item
+check_no_create_button_on_list_page:
+  action: goto_project_pods_list_page
+  action: check_yaml_create_button_missing
+  action: goto_deployment_page
+  action: check_yaml_create_button_missing
+  action: goto_serviceaccounts_page
+  action: check_yaml_create_button_missing
+  action: goto_persistentvolumeclaims_page
+  action: check_yaml_create_button_missing
+check_kebab_items_disabled_on_list_page:
```

Comment:
Can we either rename it to `check_kebab_items_disabled_on_resources_list_page` or `check_kebab_items_disabled_on_dc_list_page` ?